src/app.js

```diff
@@ -30,53 +30,54 @@ client.on('ready', () => {
 });
 
 // Create an event listener for messages
-client.on('message', msg => {
-  const { content, author, channel, member } = msg;
+client.on('message', async msg => {
+  // Ignore messages from the bot
+  if (msg.author.bot) return;
```

Comment:
I would prefer if you kept `const { content, author, channel, member } = msg;`, and used `content` instead of `msg.content`, etc.